src/dbcp/data_mart/county_concrete_mw.py

```diff
@@ -0,0 +1,111 @@
+"""Create county-level aggregates of proposed projects from EIA860m and ACP data."""
+from typing import Optional
+
+import pandas as pd
+import sqlalchemy as sa
+
+from dbcp.helpers import get_sql_engine
+
+
+def _get_concrete_aggs(engine: sa.engine.Engine) -> pd.DataFrame:
+    query = """
```

Comment:
We need to decide if we want to use SQL or python for data mart transformations. I prefer using Python because that is what our team is more comfortable with and we don't get any SQL formatting or syntax highlighting with our current method. Also, if we use pandas for transformations we can more easily switch between different storage backends.

Reply:
You're right, now I remember we had [a similar conversation](https://github.com/deployment-gap-model-education-fund/deployment-gap-model/pull/332#discussion_r1593156405) a couple months ago. I'll push this as is for the sake of time but I'll convert it to pandas next sprint